### PR TITLE
[gardening] Add copyright header. Fix \t:s and spacing. Use isa instead of dyn_cast where appropriate.

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

File: lib/AST/ASTVerifier.cpp [hunk 1]
```diff
@@ -149,7 +149,7 @@ struct LazyGenericEnvironment {
   llvm::PointerUnion<DeclContext *, GenericEnvironment *> storage;
 
   explicit operator bool() const {
-    if (storage.dyn_cast<GenericEnvironment *>())
+    if (storage.isa<GenericEnvironment *>())
```
Comment (jrose-apple) [inline lib/AST/ASTVerifier.cpp]: PointerUnion::dyn_cast isn't the same as the freestanding one.

Comment (practicalswift) [inline lib/AST/ASTVerifier.cpp]: Ouch! Thanks for noticing! Change removed.

**Comment by practicalswift** [comment]
@swift-ci please smoke test and merge

**Comment by practicalswift** [comment]
@swift-ci please test and merge

**Pull request merged**